updater.sh

```diff
@@ -42,7 +43,7 @@ retry_download() {
     local curl_retval
     local http_status
     echo "Downloading: ${url}"
-    http_status=$(curl -C- --retry 20 --fail -w "%{http_code}" -O "${url}") || curl_retval=$?
+    http_status=$(curl -C- --retry "${MAX_RETRIES}" --fail -w "%{http_code}" -O "${url}") || curl_retval=$?
```

Comment:
Given that L46 and L58 seem to be the same, is it worth breaking this out into its own function?